### PR TITLE
Update repo logo to include TypeScript logo

### PR DESCRIPTION
**Pull request by jattasNI**
Given NI's preference for TypeScript code, it seemed wrong to only show a JS logo.

I modified the logo using an image editor to include the official TS logo too.

Also updated the README to load the logo via a relative path so you can see logo changes when in a branch.

### CODE REVIEW

File: resources/logo.svg [hunk 1]
```diff
@@ -1,46 +1,44 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```
Comment (TrevorKarjanis) [inline resources/logo.svg:1]: I would suggest keeping the logos in separate files. This leaves them more easily maintainable, especially given that one is from the design team who also has a copy.

File: README.md [hunk 2]
```diff
@@ -48,4 +48,4 @@ Extend @ni/eslint-config/typescript in the ESLint configuration. Configure the @
 
 ## License
 
-[MIT (c) 2020 National Instruments Corporation](./LICENSE)
\ No newline at end of file
+[MIT (c) 2020 National Instruments Corporation](./LICENSE)
```
Comment (TrevorKarjanis) [inline README.md:51]: Should we update this to 2021?

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # NI JavaScript and TypeScript Style Guide
 
-![Logo](https://raw.githubusercontent.com/ni/javascript-styleguide/HEAD/resources/logo.svg)
+![Logo](./resources/logo.svg)
```
Comment (rajsite) [inline README.md:3]: The githubusercontent path is so the logo is not broken on pages like the npm published page: https://www.npmjs.com/package/@ni/eslint-config

See: https://github.com/ni/javascript-styleguide/pull/6/files#r508710719

Can you restore the path and add a comment to the README ie
```
<!-- Preserve the raw.githubusercontent.com path to enable image rendering when viewed from https://www.npmjs.com/package/@ni/eslint-config -->
```

**Review by rajsite — CHANGES_REQUESTED** [review]
Some additional discussion on the teams channel

File: README.md (hunk 2 above)
Comment (jattasNI) [reply README.md:51]: Good suggestion. I ended up doing this in #21

File: README.md (hunk 3 above)
Comment (jattasNI) [reply README.md:3]: Good suggestion. I ended up doing this in #21

**Comment by jattasNI** [comment]
Given the feedback in this PR and on our Teams chat, I'm going to abandon this PR and submit a request for a new logo from our design team.

**Pull request closed without merging**